src/Test/AbstractWidgetTestCase.php

```diff
@@ -89,40 +89,51 @@ protected function getTemplatePaths(): array
             // symfony/symfony (running from this bundle)
             __DIR__.'/../../vendor/symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form',
             // symfony/symfony (running from other bundles)
-            __DIR__.'/../../../../../symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form',
+            __DIR__.'/../../../../symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form',
         ], 'is_dir');
 
         $twigPaths[] = __DIR__.'/../Bridge/Symfony/Resources/views/Form';
 
         return $twigPaths;
     }
 
-    protected function getRenderingEngine(Environment $environment): TwigRendererEngine
+    protected function getRenderingEngine(Environment $environment)
     {
         return new TwigRendererEngine(['form_div_layout.html.twig'], $environment);
     }
 
     /**
      * Renders widget from FormView, in SonataAdmin context, with optional view variables $vars. Returns plain HTML.
+     *
+     * @return string
      */
-    final protected function renderWidget(FormView $view, array $vars = []): string
+    final protected function renderWidget(FormView $view, array $vars = [])
     {
         return (string) $this->renderer->searchAndRenderBlock($view, 'widget', $vars);
     }
 
     /**
      * Helper method to strip newline and space characters from html string to make comparing easier.
+     *
+     * @param string $html
+     *
+     * @return string
      */
-    final protected function cleanHtmlWhitespace(string $html): string
+    final protected function cleanHtmlWhitespace($html)
     {
-        return preg_replace_callback('/\s*>([^<]+)</', static function (array $value): string {
+        return preg_replace_callback('/\s*>([^<]+)</', static function ($value) {
```

Comment:
Why these declarations need to be removed?

Reply:
Automatic delete

Reply:
IMO, we could keep them, since those are not a break for BC.